components/Map.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import ReactMapGL from 'react-map-gl';
+
+const Map: React.FC = () => {
+  const [viewport, setViewport] = useState({
+    width: 100,
+    height: 100,
+    latitude: 41.5868,
+    longitude: -93.625,
+    zoom: 13,
+  });
+  return (
+    <ReactMapGL
+      mapStyle="mapbox://styles/mapbox/streets-v9"
+      mapboxApiAccessToken="pk.eyJ1IjoiYnJ5LWdhdmlubyIsImEiOiJja2ZydDZjdHkwOXc1MnJxcWJqNHRsejF3In0.wUyQfr7dEqtTx8Q4r2EYiA"
```

Comment:
generally, we don't want API keys to be hard-coded onto our application
move this into an environment variable

add a new key to `.env.local.development` with the prefix `NEXT_PUBLIC_` (this allows it to be accessed from the browser where this component will be rendered) so for example `NEXT_PUBLIC_MAPBOX_API_KEY`
add the same entry to `.env.example` but without the actual key so that others know what they should put in their env file